### ISSUE TITLE
Slide not working

### ISSUE THREAD

**Issue by afzaalhassan1** [afzaalhassan1/Afzaal-Websites#1]
The Javascript/Jquery part of this code is suspect and has an issue somewhere. Specifically, when I click on the arrow, it does not load the next slide/picture. The dots dont update either when the arrow is clicked. Should I redo the code for the carousel?

**Comment by shamoons** [comment]
Do you see any errors in your console??

**Comment by afzaalhassan1** [comment]
yes I see the following but I never wrote any of those lines or tried opening them in the code. Plus there is no line 371.

![screen shot 2015-05-11 at 10 55 30 pm](https://cloud.githubusercontent.com/assets/7474488/7579377/20b67aee-f831-11e4-996a-86db76b91d88.png)
